### PR TITLE
add overview one-pager

### PR DESCRIPTION
**Pull request by jimabramson**
@efagin here's how this is coming along.   @rlucioni @ahsan-ul-haq @srpearce FYI.   (still WIP)

### CODE REVIEW

**Comment by efagin** [comment]
@jimabramson looks good so far, thanks! no need to be comprehensive - freely link to other documents where appropriate. goal is to provide a simple entry point for folks trying to learn more about this IDA beyond what's noted in the [README](https://github.com/edx/credentials/blob/master/README.rst) for this repo.

**Comment by rlucioni** [comment]
You may want to link to this from the [overview](https://github.com/edx/credentials/blob/master/README.rst#overview) section of the README.

**Comment by jimabramson** [comment]
@efagin @rlucioni @ahsan-ul-haq @zubair-arbi @clintonb @srpearce made some updates to this, feedback welcome.

File: overview.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+----------------
+The service uses the user account API (hosted by the LMS) to retrieve additional information about the user, given their username.  At the moment only the full name of the user is needed.
+
+Organization Information
+------------------------
+The service uses the organizations API (also exposed by LMS) to retrieve additional information about credentialing organizations, given an ID.  The name and logo image associated with the organization are what is displayed.
+
+Program Information
+-------------------
+The service uses an API provided by the Programs IDA to get additional information about an XSeries program (e.g. name, description, number of courses) in order to display on certificates of completion for that program.
```
Comment (rlucioni) [inline overview.rst]: s/display on/display

File: overview.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+Interactions
+============
+* TODO incomplete - need to upload images
```
Comment (rlucioni) [inline overview.rst]: Looks like you did this.

File: overview.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Service Dependencies
+====================
+* edx-platform: an LMS instance is needed to host User and Organization APIs, as well as the OAuth2 / SSO provider.
+* programs IDA: hosts the Programs API which is needed to fetch metadata when rendering program-related credentials, as described above.
+
+AuthN / AuthZ
+=============
+* The edx-platform OAuth2 provider presently runs in the LMS process and can be used to initiate session-based authentication using the authorization code flow.
+* API authentication via JWT tokens is supported; the JWT tokens must be issued by the platform auth provider.
```
Comment (rlucioni) [inline overview.rst]: What about the JWTs this service signs itself and includes in requests to the LMS and programs?

**Comment by jimabramson** [comment]
thanks for the corrections @rlucioni - fixed up and rebased.

**Comment by rlucioni** [comment]
👍

**Comment by zubair-arbi** [comment]
docs looks good :+1:

**Comment by srpearce** [comment]
👍  (this can be merged while we're figuring out what to do with the credentials docs; I have a doc ticket to add the info to the docs no matter where they end up)

**Pull request merged**